index.Rmd

```diff
@@ -174,7 +174,7 @@ tm_shape(dm_pop) +
   tmap_mode("view")
 ```
 
-# Conclusions
+# Conclusions 
```

Comment:
What is the significance of 0.62? Maybe further explain this in your conclusions.